contrib/win32/openssh/OpenSSHUtils.psm1

```diff
@@ -839,7 +839,7 @@ Function Add-MachinePath {
 
     if (Test-Path $FilePath) {
         $machinePath = (Get-ItemProperty -Path 'HKLM:\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH).Path
-        if (-not $machinePath.ToLower().Contains("$FilePath;".ToLower()))
+        if (-not ($machinePath.ToLower().Contains("$FilePath;".ToLower()) -or $machinePath.ToLower().Contains("$FilePath\;".ToLower())))
```

Comment:
Instead of 'ToLower()', I feel you should use the dotNet ignore case string compare:
```powershell
"Hello".Contains('hello', [System.StringComparison]::OrdinalIgnoreCase)
```

Reply:
Does that work with Windows PowerShell? That works for me in PowerShell 7.2.7 but not Windows PowerShell 5.1

Reply:
Oh, for backwards compatibility with WindowsPowerShell you probably have to use "string".IndexOf("findstr", [System.StringComparison]::OrdinalIgnoreCase).